content/blogs/release-0-23.md

```diff
@@ -0,0 +1,442 @@
+- Switch between code and no-code approaches based on your preference or task complexity
+- Visualize task dependencies through the topology view while making edits
+- Reference documentation without leaving your editing environment
+
+You can customize your experience by opening only the panels you need, creating a workspace tailored to your workflow development style. This enhancement significantly improves productivity by reducing context switching and providing multiple perspectives on your workflows simultaneously.
+
+<div class="video-container">
+  <iframe src="https://www.youtube.com/embed/PLACEHOLDER" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+</div>
+
+### Unit Test Flows
+
+As workflows grow in complexity, so does the need to test them reliably. Kestra introduces native support for Unit Testing, giving users powerful new capabilities to validate their flows with confidence and precision.
+Until now, users relied on manual execution or external scripting to verify behavior. With Unit Test support, you can now define expected outcomes, isolate tasks, and detect regressions early—directly inside your development environment.
+
+Key components of the Unit Testing include:
+
+- **Test Suites**: Create test suites targeting individual tasks or full sections of your flows. Each suite encapsulates a set of test cases, allowing you to cover multiple paths and edge cases without executing the full logic end-to-end.
+- **Fixtures**: Define fixtures to simulate inputs or task outputs. This skips real execution for mocked tasks, making your tests faster, deterministic, and easier to isolate.
+- **Assertions**: Each test can include assertions to check for expected outcomes—task results, flow variables, execution paths, or any output produced by the flow. This helps ensure your flow behaves correctly under different conditions.
+- **API Access**: You can call the Unit Test programmatically via Kestra API, enabling automation in CI/CD pipelines, custom tooling, or integration with development workflows.
+
+
+#TODO: add example of unit test with a flow and corresponding test suite
+::collapse{title="Unit test example"}
+
+::
+
+
+<div class="video-container">
+  <iframe src="https://www.youtube.com/embed/PLACEHOLDER" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+</div>
+
+### Outputs in Internal Storage
+
+Kestra 0.23 introduces a powerful configuration-based capability to move task outputs from in-memory context (by default stored in the database) to Kestra's Internal Storage. This Enterprise Edition feature addresses critical scalability challenges for data-intensive workflows.
```

Comment:
```suggestion
By default, all inputs and outputs processed across all Kestra's workflow executions are stored in the same metadata database. However, some of our Enterprise customers require a higher degree of separation across business units (split by tenants). To address this need for separation, Kestra 0.23 introduces a powerful configuration allowing you to store all execution inputs and outputs in internal storage configured for each tenant. This way, each tenant can store that metadata in separate cloud storage buckets, enabling full data separation across tenants when needed.
```

rewritten as this section is wrong, it's not about scalability but purely about physical data separation across tenants